server/protocols/webadmin/webadmin-mailqueue/src/test/java/org/apache/james/webadmin/routes/MailQueueRoutesTest.java

```diff
@@ -505,7 +507,7 @@ public void deleteMailsShouldDeleteMailsWhenNameIsGiven() throws Exception {
         .when()
             .delete(FIRST_QUEUE + "/mails")
         .then()
-            .statusCode(HttpStatus.NO_CONTENT_204);
+            .statusCode(HttpStatus.CREATED_201);
```

Comment:
it seems you need to update the documentation (swagger and site)

Reply:
Yeah thanks, I will!